proto/ptf/base_test.py

```diff
@@ -63,9 +63,14 @@ def stringify(n, length):
     'length' bytes, it is represented in the fewest number of bytes it
     does fit into without loss of precision.  It always returns a
     string at least one byte long, even if value=width=0."""
-    h = '%x' % n
-    s = ('0'*(len(h) % 2) + h).zfill(length*2).decode('hex')
-    return s
+    if length == 0 and n == 0:
+        length = 1
+    while True:
+        try:
+            s = n.to_bytes(length, byteorder='big')
+            return s
+        except OverflowError:
+            length = length + 1
```

Comment:
As indicated in the comment, this code was equivalent to the `int.to_bytes` Python3 function, so I think we should just use that (and forget about Python2 support)

Reply:
My 2nd commit has another variation of stringify() for your consideration.  It doesn't behave exactly as int.to_bytes does alone, because I believe the Python2 version would take a parameter n that did not fit into the given number of bytes, and return a longer string with no exception.  int.to_bytes would throw an exception in that scenario.